docs/nederlands-overzicht.md

```diff
@@ -7,6 +7,54 @@ Dit kan vormgegeven zijn met een Open Source Program Office of op een andere man
 
 Zie de [Wikipedia pagina Publieke sector](https://nl.wikipedia.org/wiki/Publieke_sector) voor een indicatie van de verschillende organisaties die hieronder vallen. Dit beslaat zowel overheid als semi-overheidsorganisaties.
 
+// [:material-web:](https://www.alliander.com/nl/open-source/)
+// [:material-github:](https://github.com/MinBZK)
+
+## Ministeries
+- Ministerie van Algemene Zaken
```

Comment:
Gaan we of zullen we een sortering aanhouden? Bijvoorbeeld alfabetisch? En laten we dat dan ook documenteren bovenaan, zodat voor de lezer ook duidelijk is waarom organisaties in deze volgorde staan... (blijft toch een gevoelig punt, ben ik bang)

Reply:
Ik vul de rest wel aan met onderscheid. Kan ik gelijk rekening houden met de alfabetische volgorde 👍